NewHaven/src/GBMap.cpp

```diff
@@ -50,7 +50,7 @@ bool GBMap::placeHarvestTile(int NodeID, HarvestTile *tile) {
     // should use the operator overload
     (*board)[NodeID].tile = tile;
     *(*board)[NodeID].isPlayed = true;
-    *playCounter++;
+    *playCounter = *playCounter + 1;
     return true;

```

Comment:
`*playCounter++` doesn't work because `++` doesn't mean the same thing on pointer than it does on integers. You'll need to dereference it before using `++` this way: `(*playCounter)++` or `++(*playCounter)`.

Reply:
As seen above this has already been resolved and I know i was incrementing the pointer address. It was an accident from lack of careful attention